lib/dev/ui/spinner.rb

```diff
@@ -54,6 +54,7 @@ def check
             begin
               status = @thread.join.status
               @success = (status == false)
```

Comment:
oh yeah, it already does this if you have the task body return `false` specifically. This is a common pattern in (e.g.) ActiveRecord callbacks and such.